espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Clear.java

```diff
@@ -36,7 +37,7 @@ public Void handle(AppiumParams params) throws AppiumException {
         try {
             viewInteraction.perform(clearText());
         } catch (PerformException e) {
-            throw new AppiumException(String.format("Could not apply clear to element %s",
+            throw new InvalidElementStateException(String.format("Could not apply clear to element %s",
```

Comment:
I think it would be better to implement a new constructor ```InvalidElementStateException(String command, String elementId)```.

Reply:
these error messages are different for different cases, so I don't think such constructor may help to reduce LOC

Reply:
```"Could not apply clear to element %s"```
```"Could not click element %s"```
```"Could not apply sendKeys to element %s"```
=>
```"Could not perform %s action on element %s"```